src/components/Users/UserSummary.tsx

```diff
@@ -40,7 +40,7 @@ export default function UserSummaryTab({ userData }: { userData?: UserBase }) {
 
   const { mutate: deleteUser, isPending: isDeleting } = useMutation({
     mutationFn: async () => {
-      return await request(routes.deleteUser, {
+      return await callApi(routes.deleteUser, {
         pathParams: { username: userData?.username || "" },
       });
     },
```

Comment:
use `mutate` instead, `callApi` is not meant to be used directly in pretty much all use cases.